cypress/e2e/awx/resources/credentials.cy.ts

```diff
@@ -259,18 +264,17 @@ describe('Credentials', () => {
       cy.clickButton(/^Cancel/);
     });
   });
+
   describe('Credentials Edit: External test modal', () => {
-    let credential: Credential;
     before(() => {
       cy.createAWXCredential({
         name: 'E2E Credential ' + randomString(4),
         kind: 'Centrify Vault Credential Provider Lookup',
-        organization: organization.id,
+        organization: awxOrganization.id,
         credential_type: 25,
         inputs: { url: 'http://foo.com', client_id: 'foo', client_password: 'foo' },
       }).then((cred) => {
         credential = cred;
-        cy.giveUserCredentialsAccess(credential.name, user.id, 'Use');
```

Comment:
Intentional change? Was there any replacement to this one?

Reply:
This line was added here but was not needed in any of the tests.